packages/ace-core/src/l10n/localize.js

```diff
@@ -0,0 +1,16 @@
+const { newLocalizer } = require('@daisy/ace-localize');
+
+const enJson = require("./locales/en");
+const frJson = require("./locales/fr");
+
+export const localizer = newLocalizer({
+    en: {
+        name: "English",
```

Comment:
`name` is used in Ace GUI in the configuration dialog for user options / preferences.

Here is an example of the expanded shape of translation object, as supported by Axe (`name` and `default` are specific to Ace, not used by Axe):

```json
{
    en: {
        name: "English",
        default: true,
        translation: { hello: "Hello" },
    },
    fr: {
        name: "Français",
        translation: { hello: "Bonjour" },
    },
}
```

Typically, there are several 'localizer' instances used in an application (CLI, UI, etc.), one per package (in other words, each package supplies its own locale "database" / translated string resources). In each case, the shape of the `translation` object is totally arbitrary (but of course, expected to be consistent for all languages), whereas the `name` and `default` properties could potentially be defined by a schema. Thus why I prefer to have them at the top-level.

Reply:
`name` is used in Ace GUI in the configuration dialog for user options / preferences.

Here is an example of the expanded shape of translation object, as supported by Axe (`name` and `default` are specific to Ace, not used by Axe):

```json
{
    en: {
        name: "English",
        default: true,
        translation: { hello: "Hello" },
    },
    fr: {
        name: "Français",
        translation: { hello: "Bonjour" },
    },
}
```

Typically, there are several 'localizer' instances used in an application (CLI, UI, etc.), one per package (in other words, each package supplies its own locale "database" / translated string resources). In each case, the shape of the `translation` object is totally arbitrary (but of course, expected to be consistent for all languages), whereas the `name` and `default` properties could potentially be defined by a schema. Thus why I prefer to have them at the top-level.

Reply:
OK. The idea was to avoid having to duplicate this structure… I guess another approach could be to have that as a template object in ace-localizer, and add the `translation` property for each package… but it's may be not worth the refactoring.